src/main/resources/com/cloudbees/hudson/plugins/folder/AbstractFolder/sidepanel.jelly

```diff
@@ -22,4 +22,9 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
-<st:include it="${it.primaryView}" page="sidepanel.jelly" xmlns:st="jelly:stapler"/>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
+    <j:if test="${view == null}"> <!-- true when rendering from ModelObjectWithContextMenu.ContextMenu.from -->
+        <j:set var="view" value="${it.primaryView}"/>
+    </j:if>
+    <st:include it="${it.primaryView}" page="sidepanel.jelly"/>
```

Comment:
Is this including itself?! I mean, this file is `sidepanel.jelly` and this line is including `sidepanel.jelly`...